examples/templates/aws-linux/main.tf

```diff
@@ -155,3 +155,19 @@ resource "aws_instance" "dev" {
     Coder_Provisioned = "true"
   }
 }
+
+resource "coder_metadata" "workspace_info" {
+  resource_id = aws_instance.dev.id
+  item {
+    key   = "region"
+    value = var.region
+  }
+  item {
+    key   = "instance type"
+    value = aws_instance.dev.instance_type
+  }
+  item {
+    key   = "disk"
+    value = "${aws_instance.dev.root_block_device[0].volume_size} GB"
```

Comment:
Nitpick: I think all of these "volume size" parameters are actually measured in GiB, not decimal GB. How about labeling them that way?

Reply:
for sure!